src/server/scripts/EasternKingdoms/SunwellPlateau/boss_kalecgos.cpp

```diff
@@ -760,9 +758,8 @@ class boss_sathrovarr : public CreatureScript
                     }
                     else
                     {
-                        me->TextEmote(EMOTE_UNABLE_TO_FIND);
                         EnterEvadeMode();
-                        return;
+                        return SPELL_FAILED_TARGET_NOT_IN_INSTANCE;
```

Comment:
You cant return anything here, UpdateAI is a void function. Thats why your pull request doesnt compile. Leave it as it was before.